app/src/pages/playground/Playground.tsx

```diff
@@ -8,13 +8,14 @@ import {
   PlaygroundProvider,
   usePlaygroundContext,
 } from "@phoenix/contexts/PlaygroundContext";
+import { PlaygroundProps } from "@phoenix/store";
 
 import { PlaygroundInstance } from "./PlaygroundInstance";
 import { PlaygroundOperationTypeRadioGroup } from "./PlaygroundOperationTypeRadioGroup";
 
-export function Playground() {
+export function Playground(props: Partial<PlaygroundProps>) {
```

Comment:
nit: it might be a bit more "honest" if the "partial" state was an explicit prop like `initialState`